Source: written by Tianon Gravi, from Debian distribution
Upstream: proposed https://bugs.debian.org/831258 
Reason: FTBFS module does not compile with current gnu-efi

diff --git a/filesystems/edk2/ComponentName.h b/filesystems/edk2/ComponentName.h
index a4c3749..7a01e29 100644
--- a/filesystems/edk2/ComponentName.h
+++ b/filesystems/edk2/ComponentName.h
@@ -50,6 +50,8 @@ typedef struct _EFI_COMPONENT_NAME_PROTOCOL  EFI_COMPONENT_NAME_PROTOCOL;
                                 language specified by Language.
 
 **/
+/*
+// /usr/include/efi/efiprot.h:878:10: note: previous declaration of 'EFI_COMPONENT_NAME_GET_DRIVER_NAME' was here
 typedef
 EFI_STATUS
 (EFI_FUNCTION EFIAPI *EFI_COMPONENT_NAME_GET_DRIVER_NAME)(
@@ -57,6 +59,7 @@ EFI_STATUS
   IN  CHAR8                                *Language,
   OUT CHAR16                               **DriverName
   );
+*/
 
 
 /**
@@ -99,6 +102,8 @@ EFI_STATUS
                                 language specified by Language.
 
 **/
+/*
+// /usr/include/efi/efiprot.h:885:10: note: previous declaration of 'EFI_COMPONENT_NAME_GET_CONTROLLER_NAME' was here
 typedef
 EFI_STATUS
 (EFI_FUNCTION EFIAPI *EFI_COMPONENT_NAME_GET_CONTROLLER_NAME)(
@@ -108,6 +113,7 @@ EFI_STATUS
   IN  CHAR8                                                   *Language,
   OUT CHAR16                                                  **ControllerName
   );
+*/
 
 ///
 /// This protocol is used to retrieve user readable names of drivers 
